opencti-platform/opencti-graphql/src/manager/connectorManager.js

```diff
@@ -132,7 +132,7 @@ const connectorHandler = async () => {
 const initConnectorManager = () => {
   let scheduler;
   return {
-    start: () => {
+    start: async () => {
```

Comment:
this change seems unrelated to error/log management, is it intended ?

Reply:
Its unrelated, just a warning fix